### PR TITLE
Use 64-bit in Linux examples in docs

### PR DESCRIPTION
**Pull request by derekbruening**
Updates the example command lines in the Linux Deployment docs to use 64-bit which is far more common these days.

### CODE REVIEW

**Comment by derekbruening** [comment]
(Will confirm all failures are flakes pre-merge but almost certainly they will be for this docs tweak)

File: api/docs/deployment.dox [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* ******************************************************************************
```
Comment (abhinav92003) [inline api/docs/deployment.dox:1]: In the section on "Cross-Compiling for ARM on Linux" in building.dox, the cmake command could also be updated to use the arm64 config.

**Review by abhinav92003 — APPROVED** [review]

File: api/docs/deployment.dox (hunk 1 above)
Comment (derekbruening) [reply api/docs/deployment.dox:1]: Added a new section above it for cross-compiling for 64-bit arm.

**Comment by derekbruening** [comment]
x64 is client.attach #6452; win64 is replaceall #5412

**Pull request merged**